integration-tests/docker-compose.yaml

```diff
@@ -29,4 +29,8 @@ services:
       dockerfile: ./integration-tests/configs/prom-gen/Dockerfile
       context: ..
     ports:
-      - "9001:9001"
\ No newline at end of file
+      - "9001:9001"
+  redis:
+    image: redis:6.0.9-alpine
```

Comment:
This is a 3 year old image, maybe we want to use something newer? Especially concerning the recent news about security issues with older versions of alpine?